internal/rbd/rbd_util.go

```diff
@@ -1149,3 +1178,44 @@ func (rv *rbdVolume) ensureEncryptionMetadataSet(status string) error {
 
 	return nil
 }
+
+type snapshotInfo struct {
+	ID        int    `json:"id"`
+	Name      string `json:"name"`
+	Size      int64  `json:"size"`
+	Protected string `json:"protected"`
+	Timestamp string `json:"timestamp"`
+	Namespace struct {
+		Type         string `json:"type"`
+		OriginalName string `json:"original_name"`
+	} `json:"namespace"`
+}
+
+func (rv *rbdVolume) listSnapshots(ctx context.Context, cr *util.Credentials) ([]snapshotInfo, error) {
```

Comment:
please document a TODO like:
```
// TODO: use go-ceph once https://github.com/ceph/go-ceph/issues/300 is available in a release
```
We can then find it in [tickgit](https://www.tickgit.com/browse?repo=github.com/ceph/ceph-csi) or other tools